pkg/webhooks/server.go

```diff
@@ -255,7 +255,9 @@ func (ws *WebhookServer) handlerFunc(handler func(request *v1beta1.AdmissionRequ
 
 		admissionReview.Response = handler(request)
 		writeResponse(rw, admissionReview)
-		logger.V(3).Info("admission review request processed", "time", time.Since(startTime).String())
+		if admissionReview.Request.Kind.Group != "kyverno.io" {
```

Comment:
can we move this check earlier in the method and just not process for "kyverno.io" group? Seems odd to only check for the log...

Reply:
We need webhooks to validate/mutate clusterPolicy / policy. This condition is to hide noisy logs for Kyverno resources.

Reply:
This does not print at the default / V(2) log level right? 

If needed, I'd suggest further increasing the log level, or adding other logic to filter out unnecessary requests early, instead of making the log statement conditional as that may cause other confusion.

Reply:
The updates of policy status create lots of unnecessary logs, will find some other way to suppress it.

Reply:
In addition to logs would be good to skip the validation checks, etc. if only the policy status is updated.

Reply:
The logic is added in this PR. I'll update description once PR is ready.